utils/docker/publish_docker.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+
+set -e
+set +x
+
+trap "cd $(pwd -P)" EXIT
+cd "$(dirname "$0")"
+
+MCR_IMAGE_NAME="playwright-python"
+PW_VERSION=$(python -c "from playwright._repo_version import version;print(version)")
```

Comment:
Where does "playwright" come from? Do I need to be inside some venv for this to work? If so, sounds fragile! Any chance we can extract version to a separate file and just `cat` it here?

Reply:
Does not depend on venv. We install it globally on gha which is common practise.